src/ssa.c

```diff
@@ -401,6 +401,54 @@ void build_rdf()
     free(args);
 }
 
+void users_add_tail(insn_t *i, var_t *var)
```

Comment:
Is `use_chain_add_tail` or `uc_add_tail` better for understanding?

Reply:
I think `use_chain_add_tail` is better; I'll change change it to that.